### PR TITLE
Add how to create bastion server

### PR DESCRIPTION
**Pull request by kota2and3kan**
This PR adds a document `Create a bastion server`. Basically, we use the `kubectl` and `helm` commands for the deployment of Scalar products.

Please take a look!

### CODE REVIEW

File: docs/CreateBastionServer.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+This document explains how to create a bastion server and install some tools for the deployment of Scalar products.
+
+## Create a server on the same private network as a Kubernetes cluster
+
+It is recommended to create a Kubernetes cluster for Scalar products on a private network. If you create a Kubernetes cluster on a private network, you should create a bastion server on the same private network to access your Kubernetes cluster.
+
+## Install tools
+
+Please install the following two tools on the bastion server according to each official document.
```
Comment (feeblefakie) [inline docs/CreateBastionServer.md:11]:
```suggestion
Please install the following tools on the bastion server according to their official documents.
```

File: docs/CreateBastionServer.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+After you install the kubectl command, you must configure a **kubeconfig** to access your Kubernetes cluster. Please refer to the following official document for more details on how to configure kubeconfig in each managed Kubernetes.
+
+If you use Amazon EKS (Amazon Elastic Kubernetes Service), you must install the **AWS CLI** according to the official document [Installing or updating the latest version of the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html). After that, you can see how to configure kubeconfig in [Creating or updating a kubeconfig file for an Amazon EKS cluster](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html).
+
+If you use AKS (Azure Kubernetes Service), you must install the **Azure CLI** according to the official document [How to install the Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli). After that, you can see how to configure kubeconfig in [az aks get-credentials](https://learn.microsoft.com/en-us/cli/azure/aks?view=azure-cli-latest#az-aks-get-credentials).
+
+## Check installation
+
+You can check if tools are installed as follows.
```
Comment (feeblefakie) [inline docs/CreateBastionServer.md:26]:
```suggestion
You can check if the tools are installed as follows.
```

File: docs/CreateBastionServer.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+* kubectl
+  ```console
+  kubectl version --client
+  ```
+* helm
+  ```console
+  helm version
+  ```
+
+Also, you can check if your kubeconfig is properly configured as follows. If you see a URL response, kubectl is correctly configured to access your cluster.
```
Comment (feeblefakie) [inline docs/CreateBastionServer.md:37]:
```suggestion
You can also check if your kubeconfig is properly configured as follows. If you see a URL response, kubectl is correctly configured to access your cluster.
```

**Review by feeblefakie — COMMENTED** [review]
Overall, looking good. Thank you!
Left minor suggestions, so PTAL when you get a chance.

**Comment by kota2and3kan** [comment]
@feeblefakie  
Thank you for your suggestion!
I applied them. Please take a look when you have time.

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**